handlers_test.go

```diff
@@ -5,15 +5,15 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"phoval"
-	"phoval/storage"
+	"phoval/mock"
```

Comment:
Where is the circular dependency? 
I don't see the package `phoval/mock`. Anyway I mean to create a `mock` package..